
Some hackery to get mips building.

  - Rene Rebe <rene@exactcode.de>

--- pdksh-5.2.14/sh.h.vanilla	2005-11-06 16:30:50.000000000 +0100
+++ pdksh-5.2.14/sh.h	2005-11-06 16:31:18.000000000 +0100
@@ -251,7 +251,7 @@
 # if SIZEOF_LONG >= 4
 #  define INT32	long
 # else /* SIZEOF_LONG */
-   #error cannot find 32 bit type...
+#  define INT32 int32_t
 # endif /* SIZEOF_LONG */
 #endif /* SIZEOF_INT */
 
